README.md

```diff
@@ -105,27 +103,23 @@ There are multiple ways -
 
 ## Contributing
 
-Bug reports and pull requests are welcome on GitHub at [alacritty-theme](https://github.com/eendroroy/alacritty-theme)
-repository. This project is intended to be a safe, welcoming space for collaboration, and contributors are expected to
-adhere to the [Contributor Covenant](http://contributor-covenant.org) code of conduct.
-
-  1. Fork it ( https://github.com/eendroroy/alacritty-theme/fork )
-  2. Create your feature branch (`git checkout -b my-new-feature`)
-  3. Commit your changes (`git commit -am 'Add some feature'`)
-  4. Push to the branch (`git push origin my-new-feature`)
-  5. Create a new Pull Request
-  
-  Suggestions:
-  
-  - Ensure `{theme}.yaml` file in `theme` directory
-  - Ensure theme preview (ie: `{theme}.png`) in `images` directory
-  - Use [print_colors](https://raw.githubusercontent.com/eendroroy/bin_scripts/master/public/print_colors) script to generate preview
-  - Ensure `schemes.yaml` file is updated
-  - Ensure theme listing in `README.md` following alphabetical ordering
-
-## Author
-
-* **indrajit** - *Owner* - [eendroroy](https://github.com/eendroroy)
+Bug reports and pull requests are welcome on GitHub at the [alacritty-theme]
+repository.
+
+[alacritty-theme]: https://github.com/alacritty/alacritty-theme
+
+To add a new theme, just create a Pull Request with the following changes:
+
+ - Add your theme to the `themes` directory with the `{theme}.yaml` file format
+ - Create a screenshot of your theme using the [print_colors script]
+ - Add the screenshot to the `immages` directory with the `{theme}.png` file format
+ - Add your theme to the `schemes.yaml`
+ - Add your theme to the `README.md`, following alphabetical ordering
+
+## Maintainers
+
+ * **indrajit** - *Author* - [eendroroy](https://github.com/eendroroy)
+ * **Christian Dürr** - *Maintainer* - [chrisduerr](https://github.com/chrisduerr)
```

Comment:
I've kept this section and just added myself, but especially with a project that has so many collaborators it's maybe a bit questionable if this is necessary at all. It doesn't really provide much value to users imo but I didn't want to take away your credit in creating this project.